tests/systemd_testsuite/binary_tests.pm

```diff
@@ -0,0 +1,40 @@
+# SUSE's openQA tests
+#
+# Copyright © 2019 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: Run test executed by TEST-01-BASIC from upstream after openSUSE/SUSE patches.
+# Maintainer: Sergio Lindo Mansilla <slindomansilla@suse.com>, Thomas Blume <tblume@suse.com>
+
+use base "consoletest";
+use warnings;
+use strict;
+use testapi;
+use utils 'zypper_call';
+use power_action_utils 'power_action';
+use systemd_testsuite_test qw(testsuiteinstall faillog);
+
+sub run {
+    testsuiteinstall;
+
+    #run binary tests
+    assert_script_run 'cd /var/opt/systemd-tests';
+    assert_script_run './run-tests.sh | tee /tmp/testsuite.log', 600;
+    assert_screen("systemd-testsuite-binary-tests-summary");
+}
+
+sub test_flags {
+    return {milestone => 1};
+}
+
+sub post_fail_hook {
```

Comment:
We can remove this implementation in all childs when it is on the parent class.

Reply:
Ok, I have updated the patch, can you take a look again?

Reply:
Verification run: http://emerson.suse.de/tests/912